setup.py

```diff
@@ -40,7 +40,7 @@ def load_about():
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     python_requires=">=3.5",
-    install_requires=["tutor>=14.0.0,<15.0.0"],
+    install_requires=["tutor>=15.0.0,<16.0.0"],
```

Comment:
For the same reason we don't bump the `__about__` version, we must not bump these.
```suggestion
    install_requires=["tutor>=14.0.0,<15.0.0"],
```

Reply:
Again to be clear guys, after this gets merged into `nightly` people who be intrested to test this, should also or are expected to use tutor `nightly` or `olive` , but however if they install this way, it will force them for tutor-14, which might incldue new patch/update about patches priorities thus the installtion might fail. Make I am wrong about this?

Reply:
@ghassanmas a tutor plugin should be tested using the corresponding tutor branch. Because we are merging this change into`tutor-mfe/nightly`, it will be tested alongside `tutor/nightly`. Nightly has the latest version of all features, including filter priorities. I know this is a little confusing because its version number is still 14.x, but it makes sense if you consider that Nightly is *"master (Nutmeg) plus new things"*, not *"the next release (Olive) plus new things"*.

When the change is merged into `tutor-mfe/olive` (@arbrandes , not sure if you will do this immediately, or wait until further testing), then it will be tested alongside `tutor/olive`, and all the version numbers will be 15.x. That version bump will be handled by commits on the `olive` branches, so this PR does not need to worry about bumping the version number.

Reply:
I don't recommend merging it immediately, no.  I think we should use nightly to get the feature tested more widely, presumably because some people already have nightly installations going.

As for the actual version number bump, I _think_ it will only come when the olive branch is merged into master, so immediately prior to release.  This is because tutor itself must be version-bumped _first_.